src/xdocs/config_whitespace.xml

```diff
@@ -600,21 +600,68 @@ class Test {
       </subsection>
 
       <subsection name="Examples" id="FileTabCharacter_Examples">
-        <p>
-          To configure the check to report on the first instance in each
-          file:
-        </p>
+        <p>To configure the check to report only the first instance in each file:</p>
         <source>
 &lt;module name=&quot;FileTabCharacter&quot;/&gt;
         </source>
-        <p>
-          To configure the check to report on each line in each file:
-        </p>
+        <p>Example:</p>
+        <source>
+public class test {
```

Comment:
`class Test`

Reply:
make new paragrath above `source` and define name of fie "Test.java".

Reply:
Done

Reply:
Done